universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -149,8 +149,8 @@
               </else>
             </choose>
           </if>
-          <else-if type="thesis">
-            <text variable="genre" suffix="."/>
+          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
+            <text variable="note" suffix="."/>
```

Comment:
@edhek007, according to Mendeley's documentation, the "Thesis" item type should have a "Genre" field. See the entry "TYPE: Thesis -> thesis" at http://support.mendeley.com/customer/portal/articles/364144-csl-type-mapping. Is that really not the case?